pipelines/appco/scripts/longhorn-setup.sh

```
@@ -0,0 +1,623 @@
+  if [[ ! -z ${CUSTOM_LONGHORN_CSI_SNAPSHOTTER_IMAGE} ]]; then
+    sed -i 's#'${LONGHORN_CSI_SNAPSHOTTER_IMAGE}'#'${CUSTOM_LONGHORN_CSI_SNAPSHOTTER_IMAGE}'#' "${MANIFEST_BASEDIR}/longhorn.yaml"
+  else
+    # use csi snapshotter image specified in yaml file if custom image is not specified
+    CUSTOM_LONGHORN_CSI_SNAPSHOTTER_IMAGE=${LONGHORN_CSI_SNAPSHOTTER_IMAGE}
+  fi
+
+  if [[ ! -z ${CUSTOM_LONGHORN_CSI_LIVENESSPROBE_IMAGE} ]]; then
+    sed -i 's#'${LONGHORN_CSI_LIVENESSPROBE_IMAGE}'#'${CUSTOM_LONGHORN_CSI_LIVENESSPROBE_IMAGE}'#' "${MANIFEST_BASEDIR}/longhorn.yaml"
+  else
+    # use csi liveness probe image specified in yaml file if custom image is not specified
+    CUSTOM_LONGHORN_BACKING_IMACUSTOM_LONGHORN_CSI_LIVENESSPROBE_IMAGEGE_MANAGER_IMAGE=${LONGHORN_BACKING_IMAGE_MANAGER_IMAGE}
```

Shellcheck (0.10.0)

[warning] CUSTOM_LONGHORN_BACKING_IMACUSTOM_LONGHORN_CSI_LIVENESSPROBE_IMAGEGE_MANAGER_IMAGE appears unused. Verify use (or export if used externally).

(SC2034)